src/cmd/linuxkit/pkglib/docker.go

```diff
@@ -39,13 +39,13 @@ func (dr dockerRunner) command(args ...string) error {
 	if dr.dct {
 		cmd.Env = append(cmd.Env, dctEnableEnv)
 	}
-	if debugDockerCommands {
-		var dct string
-		if dr.dct {
-			dct = dctEnableEnv + " "
-		}
-		fmt.Fprintf(os.Stderr, "+ %s%v\n", dct, cmd.Args)
+
+	dct := ""
+	if dr.dct {
+		dct = dctEnableEnv + " "
```

Comment:
Unless the diff is misleading me with the removal of the `debugDockerCommands` conditional it now makes sense to merge the to `if dr.dct` blocks I think.